src/main/java/com/softserve/actent/resources/StringConstants.java

```diff
@@ -0,0 +1,20 @@
+package com.softserve.actent.resources;
```

Comment:
This file should not be .java, better to use .properties for messages 

check it out
https://www.baeldung.com/spring-boot-internationalization

Reply:
I don't create this folder. But can create .properties file

Reply:
but in your merge request seems like you've added all the lines, at least from your account ;)

Reply:
@Pitcity i pull this class and add code after "// USER" comment. And i definetely would not call the package "resources" :)) But why static final strings in class is a bad idea for a constants ?